frontend/src/App.js

```diff
@@ -4,7 +4,10 @@ import { SharedStateProvider } from './Context/SharedStateContext';
 import { useAuthContext } from './hooks/useAuthContext';
 import Home from './pages/Home';
 import TaskDetails from './pages/TaskDetails';
-import Groups from './pages/Groups';
+import GroupPage from './pages/GroupPage';
+
```

Comment:
remove line

Reply:
I have enabled that extension, and it seems to be running. Is there any additional setup I might have missed?

Reply:
you need to use for me its "option + shift + f" to actually format the code.